apps/openmw/mwphysics/physicssystem.cpp

```diff
@@ -1152,9 +1152,9 @@ namespace MWPhysics
 
                 bool waterCollision = false;
                 if (effects.get(ESM::MagicEffect::WaterWalking).getMagnitude()
-                        && cell->getCell()->hasWater()
-                        && !world->isUnderwater(iter->first.getCell(),
-                                               osg::Vec3f(iter->first.getRefData().getPosition().asVec3())))
+                        && cell->getCell()->hasWater())
+//                        && !world->isUnderwater(iter->first.getCell(),
+//                                               osg::Vec3f(iter->first.getRefData().getPosition().asVec3())))
```

Comment:
You're not alone!  We're very welcoming to do new developers and we try to help guide them, like we are doing with you. Getting it right the first time pays off in the long run, and you're doing well otherwise! Great job!

Feel free to close this PR since you've opened a new one.